### PR TITLE
Reviews

### PR DESCRIPTION
**Pull request by raphaelspies**
basic functionality for Reviews:
-Review and ReviewList components
-ReviewList maps Reviews
-temporary Axios calls directly to atelier API for getReviews and getProducts
-Product id is temporarily hardcoded

### CODE REVIEW

File: client/src/components/ReviewList.jsx [hunk 1]
```diff
@@ -1,16 +1,60 @@
...
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import credentials from '../../../config.js'
 
 class ReviewList extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-      reviews: []
+      reviews: [],
```
Comment (erikeh) [inline client/src/components/ReviewList.jsx:12]: I believe there should be no comma for a single property

Comment (erikeh) [reply client/src/components/ReviewList.jsx:12]: Just saw new state - last property of object should not have comma i believe

**Review by erikeh — APPROVED** [review]
Looks good. just made a note about commas in the state

**Comment by raphaelspies** [comment]
> Looks good. just made a note about commas in the state

I'm not super picky about the commas, but I saw Dan Abramov put commas after every item in his objects/arrays when he was doing the React Hooks demo and I thought it was a cool way to avoid forgetting commas later when you add items

**Comment by erikeh** [comment]
> > Looks good. just made a note about commas in the state
> 
> I'm not super picky about the commas, but I saw Dan Abramov put commas after every item in his objects/arrays when he was doing the React Hooks demo and I thought it was a cool way to avoid forgetting commas later when you add items

Didn't know about that! if Dan Abramov does it, i think it's safe to assume it's chill.

File: client/src/components/ReviewList.jsx [hunk 2]
```diff
@@ -1,16 +1,91 @@
...
+  componentDidMount() {
+    this.getProductList()
+    this.getReviews(16057)
+  }
+
+  getProductList() {
+    axios({
+      method: 'GET',
+      url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-lax/products/',
+      headers: { 'Authorization': credentials.TOKEN },
```
Comment (Lordnickim) [inline client/src/components/ReviewList.jsx:28]: Last comma in headers might throw something

Comment (Lordnickim) [reply client/src/components/ReviewList.jsx:28]: Also were planning on doing the Api requests via server later right? Might need to change later

**Review by Lordnickim — APPROVED** [review]

File: client/src/components/ReviewList.jsx (hunk 2 above)
Comment (raphaelspies) [reply client/src/components/ReviewList.jsx:28]: yes--will definitely refactor later to do API calls to server.

**Pull request merged**